### ISSUE TITLE
help need

### ISSUE THREAD

**Issue by adnanaslam475** [Saad-Bashar/LocalPush-Notifications#1]
if we set daily notification at  2pm daily? would it be run even if app is not running in background?

**Comment by Saad-Bashar** [comment]
yes, the app does not need to be running. Once we schedule a notification, it will appear at the exact time even if the app is not running.

**Comment by adnanaslam475** [comment]
great, it doesnt need firebase fcm, right?

**Comment by Saad-Bashar** [comment]
Yes, correct. It does not need FCM. You can set it from your side.

**Comment by adnanaslam475** [comment]
jazakallah. let me check

**Comment by adnanaslam475** [comment]
I am getting this error on first run
![image](https://user-images.githubusercontent.com/38522696/132121305-7ca6c6e8-10a7-43cb-a172-6b67f482204b.png)

**Comment by Saad-Bashar** [comment]
Not sure what you are doing but if you see my repo, there is no firebase and it's working fine for local notification

**Comment by 3beeepb** [comment]
same problem

**Comment by 3beeepb** [comment]
works for me
requestPermissions:Platform.OS === 'ios'

**Comment by shourov-sorkar** [comment]
**There is no firebase in my app, but why I am getting this error?**

![image](https://user-images.githubusercontent.com/61703809/179477257-8d76ddab-70e5-4c02-89a8-aebaefb67b77.png)

**Comment by msriaz** [comment]
if you are not using remote notification or do not have Firebase installed, use this:
       `requestPermissions: Platform.OS === 'ios'`
      
https://github.com/zo0r/react-native-push-notification/issues/1763

@Saad-Bashar I have created a PR for this fix. Please merge [this PR.](https://github.com/Saad-Bashar/LocalPush-Notifications/pull/2)
